### PR TITLE
add endpoints to help debug

### PR DESCRIPTION
**Pull request by frederic34**

### CODE REVIEW

File: zb-classifier.js [hunk 1]
```diff
@@ -1727,16 +1727,36 @@ class ZigbeeClassifier {
...
     const genLevelCtrlOutputEndpoints =
-        node.findZhaEndpointWithOutputClusterIdHex(CLUSTER_ID.GENLEVELCTRL_HEX);
+      node.findZhaEndpointWithOutputClusterIdHex(CLUSTER_ID.GENLEVELCTRL_HEX);
     const genOnOffEndpoints =
-        node.findZhaEndpointsWithInputClusterIdHex(CLUSTER_ID.GENONOFF_HEX);
+      node.findZhaEndpointsWithInputClusterIdHex(CLUSTER_ID.GENONOFF_HEX);
     const genOnOffOutputEndpoints =
-        node.findZhaEndpointWithOutputClusterIdHex(CLUSTER_ID.GENONOFF_HEX);
+      node.findZhaEndpointWithOutputClusterIdHex(CLUSTER_ID.GENONOFF_HEX);
+    const closureShadeCfgEndpoints =
```
Comment (mrstegeman) [inline zb-classifier.js:1746]: These are named `CLOSURES*` below, i.e. pluralized.

**Review by mrstegeman — CHANGES_REQUESTED** [review]

File: zb-constants.js [hunk 2]
```diff
@@ -145,6 +146,8 @@ const CLUSTER_ID = {
   RELATIVE_HUMIDITY: zclId.cluster('msRelativeHumidity').value,
   SEMETERING: zclId.cluster('seMetering').value,
   SSIASZONE: zclId.cluster('ssIasZone').value,
+  SSIASACE: zclId.cluster('ssIasAce').value,
```
Comment (mrstegeman) [inline zb-constants.js:149]: What are these 2 new cluster IDs for? They're not used within the other changes.

Comment (frederic34) [reply zb-constants.js]: I remove them, I will make other tests later

**Review by mrstegeman — APPROVED** [review]

**Comment by mrstegeman** [comment]
@frederic34 Anything else you're wanting to merge, or should I build a release?

**Comment by frederic34** [comment]
@mrstegeman I made some test, seems good, nothing to add for now, I have to work with closures
`2020-11-17 22:02:24.237 INFO   : zigbee-adapter: classify called for node: ec1bbdfffed2cde0
2020-11-17 22:02:24.238 INFO   : zigbee-adapter: ---- Zigbee classifier -----
2020-11-17 22:02:24.239 INFO   : zigbee-adapter:                              modelId = FB56+CUR17SB2.2
2020-11-17 22:02:24.239 INFO   : zigbee-adapter:                   seMeteringEndpoint = undefined
2020-11-17 22:02:24.240 INFO   : zigbee-adapter:                 haElectricalEndpoint = undefined
2020-11-17 22:02:24.241 INFO   : zigbee-adapter:               genBinaryInputEndpoint = undefined
2020-11-17 22:02:24.242 INFO   : zigbee-adapter:                   genScenesEndpoints = [ 1 ]
2020-11-17 22:02:24.242 INFO   : zigbee-adapter:             genScenesOutputEndpoints = []
2020-11-17 22:02:24.243 INFO   : zigbee-adapter:                   genGroupsEndpoints = [ 1 ]
2020-11-17 22:02:24.244 INFO   : zigbee-adapter:             genGroupsOutputEndpoints = []
2020-11-17 22:02:24.245 INFO   : zigbee-adapter:                 genLevelCtrlEndpoint = undefined
2020-11-17 22:02:24.246 INFO   : zigbee-adapter:          genLevelCtrlOutputEndpoints = []
2020-11-17 22:02:24.246 INFO   : zigbee-adapter:                    genOnOffEndpoints = [ 1 ]
2020-11-17 22:02:24.247 INFO   : zigbee-adapter:              genOnOffOutputEndpoints = []
2020-11-17 22:02:24.248 INFO   : zigbee-adapter:            closuresShadeCfgEndpoints = []
2020-11-17 22:02:24.249 INFO   : zigbee-adapter:      closuresShadeCfgOutputEndpoints = []
2020-11-17 22:02:24.249 INFO   : zigbee-adapter:      closuresWindowCoveringEndpoints = [ 1 ]
2020-11-17 22:02:24.250 INFO   : zigbee-adapter: losuresWindowCoveringOutputEndpoints = []
2020-11-17 22:02:24.251 INFO   : zigbee-adapter:               hvacFanControlEndpoint = undefined
2020-11-17 22:02:24.252 INFO   : zigbee-adapter:               hvacThermostatEndpoint = undefined
2020-11-17 22:02:24.252 INFO   : zigbee-adapter:                     doorLockEndpoint = undefined
2020-11-17 22:02:24.253 INFO   : zigbee-adapter:              lightingControlEndpoint = undefined
2020-11-17 22:02:24.254 INFO   : zigbee-adapter:                    colorCapabilities = undefined
2020-11-17 22:02:24.255 INFO   : zigbee-adapter:                            colorMode = undefined
2020-11-17 22:02:24.256 INFO   : zigbee-adapter:                    lightLinkEndpoint = undefined
2020-11-17 22:02:24.256 INFO   : zigbee-adapter:           msOccupancySensingEndpoint = undefined
2020-11-17 22:02:24.257 INFO   : zigbee-adapter:                msTemperatureEndpoint = undefined
2020-11-17 22:02:24.258 INFO   : zigbee-adapter:                  genPowerCfgEndpoint = undefined
2020-11-17 22:02:24.259 INFO   : zigbee-adapter:             genDeviceTempCfgEndpoint = undefined
2020-11-17 22:02:24.259 INFO   : zigbee-adapter:                             zoneType = undefined
2020-11-17 22:02:24.260 INFO   : zigbee-adapter: genOnOffEndpoints = [ 1 ]
2020-11-17 22:02:24.261 INFO   : zigbee-adapter: Processing endpoing 0 = 1
2020-11-17 22:02:24.262 INFO   : zigbee-adapter: addProperty: ec1bbdfffed2cde0 on EP:1 CL:0006
2020-11-17 22:02:24.262 INFO   : zigbee-adapter: addProperty:    fireAndForget: false bindNeeded: true configReportNeeded: true initialReadNeeded: true
2020-11-17 22:02:24.263 INFO   : zigbee-adapter: classify: Initialized as type: switch name: zb-ec1bbdfffed2cde0-switch defaultName: zb-ec1bbdfffed2cde0-switch
`

**Pull request merged**